sdk/src/main/java/com/google/cloud/dataflow/sdk/io/CompressedSource.java

```diff
@@ -356,14 +356,23 @@ public final DecompressingChannelFactory getChannelFactory() {
 
     private final FileBasedReader<T> readerDelegate;
     private final CompressedSource<T> source;
+    private final boolean splittable;
```

Comment:
Java convention is that `isSplittable` is the getter for a boolean named splittable.